dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ResourcesServiceImpl.java

```diff
@@ -872,9 +872,10 @@ public Map<String, Object> queryResourceList(User loginUser, ResourceType type,
 
         Tenant tenant = tenantMapper.queryById(user.getTenantId());
         if (tenant == null) {
-            log.error("tenant not exists");
-            putMsg(result, Status.CURRENT_LOGIN_USER_TENANT_NOT_EXIST);
-            return null;
```

Comment:
I think we should keep this error msg, which can tell user the real reason.

Reply:
Done.